### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by Constant1ne-0819** [aclap-dev/video-downloadhelper#500] [Help and Q&A]
### Your question

I got the message Download failed so please help me to fix it

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://watch.shelter.stream/the-promise/videos/the-promise-architect-bv-doshi",
  "error": {
    "details": "SideDownload error: : [hls @ 0000017761897880] Unable to open key file skd://drm\r\n[aac @ 0000017763657340] Number of bands (48) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 29 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[h264 @ 0000017763669400] top block unavailable for requested intra mode -1\r\n[h264 @ 0000017763669400] error while decoding MB 6 0, bytestream 18939\r\n[aac @ 0000017763657340] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (61) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] skip_data_stream_element: Input buffer exhausted before END element found\r\n[aac @ 0000017763657340] error in spectral data, ESC overflow\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (64) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (63) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (75) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (75) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (54) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] channel element 0.11 is not allocated\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (66) exceeds limit (46).\r\n[aac @ 0000017763657340] Input buffer exhausted before END element found\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (73) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] error in spectral data, ESC overflow\r\n[aac @ 0000017763657340] Number of bands (69) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (63) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (66) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] Scalefactor (-5) out of range.\r\n[aac @ 0000017763657340] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (66) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (75) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 29 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 29 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (61) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 2 times\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (60) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (48) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (67) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (65) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (48) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (64) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 30 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 29 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (60) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (47) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (68) exceeds limit (46).\r\n[aac @ 0000017763657340] error in spectral data, ESC overflow\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] channel element 2.1 is not allocated\r\n[aac @ 0000017763657340] TNS filter order 30 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (66) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (69) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (60) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (48) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 30 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (60) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (71) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (63) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] channel element 1.14 is not allocated\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (54) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (56) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (52) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (66) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (60) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (53) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] TNS filter order 26 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (49) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (54) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (61) exceeds limit (46).\r\n[aac @ 0000017763657340] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 0000017763657340] Number of bands (58) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (51) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (57) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (54) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 0000017763657340] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[aac @ 0000017763657340] Number of bands (18) exceeds limit (13).\r\n[aac @ 0000017763657340] Number of bands (59) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (62) exceeds limit (46).\r\n[aac @ 0000017763657340] Number of bands (50) exceeds limit (46).\r\n[aac @ 0000017763657340] invalid band type\r\n[aac @ 0000017763657340] Number of bands (55) exceeds limit (46).\r\n[aac @ 0000017763657340] Pulse data corrupt or invalid.\r\n[mp4 @ 0000017763657740] sample rate not set\r\n[out#0/mp4 @ 0000017763642800] Could not write header (incorrect codec parameters ?): Invalid argument\r\nError opening output file C:\\Users\\doant\\Downloads\\The Promise - The Promise - Shelter-01.mp4.\r\nError opening output files: Invalid argument\r\n",
    "downloadable_id": "downloadable_4440555724823311",
    "error": "coapp_failure",
    "id": "coapp_failure_42a7bfaa-f778-48ac-9f1d-b7573e968158"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I visited the URL you provided on watch.shelter.stream.  The page says I need to register to watch the content.  I am not going to register to watch the content.

I can't help you.

**Comment by Mattmjs** [comment]
I'm guessing based on such a large error message that the video is protected with DRM, you can verify this using Firefox.
In Firefox , not chrome or edge. Do you see this icon :

![DRM icon](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/a17a447d-9be2-4ec8-a9b9-0d34f28a78b8)
